Sources/EdgeNetworkHandlers/RequestBuilder.swift

```diff
@@ -23,8 +23,8 @@ class RequestBuilder {
     /// Control character used at the end of each response fragment. Response streaming is enabled when both `recordSeparator` and `lineFeed` are non nil.
     private var lineFeed: String?
 
-    /// The Experience Cloud ID to be sent with this request
-    var experienceCloudId: String?
+    /// The`IdentityMap` to be attached to the request
+    var identityMap: IdentityMap?
```

Comment:
As in my other comment, I would prefer to keep the Identity map as raw dictionary